resources/carrier/en/98.txt

```diff
@@ -21,10 +21,18 @@
 98931|MTCE
 98932|Taliya
 98933|Irancell
-98934|TKC
+98934|TeleKish
 98935|Irancell
 98936|Irancell
 98937|Irancell
 98938|Irancell
 98939|Irancell
 98990|IR-MCI
+98991|IR-MCI
+9899810|Shatel Mobile(Full MVNO)
```

Comment:
If you look at Numbering Plan File(Mine=ITU+Operators List or Directly from ITU), you will see 99810 instead of 998.
In addition, shatel mobile shows only numbers starting with 99810 when you want to register for a new number.
You can check from here: https://my.shatelmobile.ir/shatelclients/wicket/page?3 (You can change language to English if it was Persian, by click on UK flag in page)
But as you saw, the news in their official website saying 998. It's because they aquired licence of 99810 for start of working, but they will add to range(They will aquire licence from regulatory) if for more numbers in 998.
It's an advertsing technique for them that anouncing 998 from first step. We should look at reallities, not advertisings. One of important reallities is ITU document that showing 99810 and onother one is registeration system in their offiicial website that registering for 99810 number for now.
If they accquired all of 998 or started to registration of 9982 or 9983, i will add it to libphone number in future.
We should keep libphonenumber synced with current status not pllaned status of operators.